http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/BaseCloudServiceRuntimeMonitor.scala

```diff
@@ -90,7 +93,68 @@ abstract class BaseCloudServiceRuntimeMonitor[F[_]] {
                     errorDetails: Option[RuntimeErrorDetails],
                     instances: Set[DataprocInstance])(
     implicit ev: ApplicativeAsk[F, AppContext]
-  ): F[CheckResult]
+  ): F[CheckResult] =
+    for {
+      ctx <- ev.ask
+      _ <- List(
+        // Delete the cluster in Google
+        runtimeAlg
+          .deleteRuntime(
+            DeleteRuntimeParams(
+              runtimeAndRuntimeConfig.runtime.googleProject,
+              runtimeAndRuntimeConfig.runtime.runtimeName,
+              runtimeAndRuntimeConfig.runtime.asyncRuntimeFields.isDefined
+            )
+          )
+          .void, //TODO is this right when deleting or stopping fails?
+        //save cluster error in the DB
+        saveRuntimeError(
+          runtimeAndRuntimeConfig.runtime.id,
+          errorDetails,
+          None //TODO: pass in startup script path in the future
```

Comment:
This was `None` for both GCE and Dataproc, so I left this TODO here in the base class

Reply:
hmm, can't remember why I left the TODO anymore...I don't mind if we just remove it either...

Reply:
Yeah.. it looks like this was mainly for reporting user script errors in the Dataproc case (the GCE case covers this). Maybe will see if I can fix this easily, and remove the TODO